workshop.wwwapi/Models/Payloads.cs

```diff
@@ -0,0 +1,6 @@
+namespace workshop.wwwapi.Models
```

Comment:
these belong to endpoitns / api, better moved in that folder, not the models folder